solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -537,10 +537,17 @@ contract RandomBeaconGovernance is Ownable {
         external
         onlyOwner
     {
+        (, uint64 authorizationDecreaseDelay, ) = randomBeacon
+            .authorizationParameters();
+        // In blocks assuming 15s block time
+        uint256 groupLifetimeUpperBoundary = authorizationDecreaseDelay / 15;
+        // 1 day assuming 15s block time
+        uint256 groupLifetimeLowerBoundary = 5760;
         /* solhint-disable not-rely-on-time */
         require(
-            _newGroupLifetime >= 1 days && _newGroupLifetime <= 2 weeks,
-            "Group lifetime must be >= 1 day and <= 2 weeks"
+            _newGroupLifetime >= groupLifetimeLowerBoundary &&
+                _newGroupLifetime <= groupLifetimeUpperBoundary,
+            "Group lifetime is not in range"
```

Comment:
> That being said, I do not think we should hardcode in the code 15s as an average Ethereum block time.
I think we should validate if it is non-zero, and add a note to @notice section [..]

The thing is that `_newGroupLifetime` is in blocks but `authorizationDecreaseDelay` is in seconds and we also compared blocks of `_newGroupLifetime` to time. 
Not sure if I understand your comment above right but are you proposing to just validate if the `_newGroupLifetime > 0`? If yes, we can do that, especially since the average block time is fluid. Besides, this is the governance update and they should be aware of what values to put there. I would also add a note to `@notice` that `authorizationDecreaseDelay` is in seconds and the `_newGroupLifetime` blocks need to be calculated accordingly. wdyt?